### DISCUSSION TITLE
Split Copper Block textures into their own files

### DISCUSSION THREAD

**Discussion by MagmaSlime123** [Roadhog360/Et-Futurum-Requiem#71] [Ideas]
More of a suggestion, but all the Copper Blocks/Slabs/Stairs/Cut Blocks (Normal and its oxidized versions) should have their own texture files rather then being tied to 1 texture file.

I was gonna retexture the waxed versions to add a bit of a honey-yellow-y-ness to them to show the wax-ness, but due to all the blocks being tied to the default Copper Block file for each block, then I can't.

**Comment by MagmaSlime123** [comment]
If you won't add this then how can Waxed versions be changed.

**Reply by Roadhog360** [reply]
They don't.
